project1_tiny_calculator/lib/main.dart

```diff
@@ -176,7 +133,7 @@ class _TinyCalculatorState extends State<TinyCalculator> {
               ),
               child: Center(
                 child: Text(
-                  'Result : $_result',
+                  'Result : ${provider.result}',
```

Comment:
```suggestion
                  provider.result,
```

original design does not have the word Result

Reply:
done